frontend/src/index.tsx

```diff
@@ -22,13 +23,15 @@ doLogin().then(() => {
     tags.setTagList().then(() => {
       species.setSpecieList().then(() => {
         sexes.setSexes().then(() => {
-          phenotypes.setPhenotypes().then(() => {
-            projections.setProjections().then(() => {
-              root.render(
-                <Provider store={store}>
-                  <App />
-                </Provider>
-              );
+          populations.setPopulations().then(() => {
```

Comment:
Not related with this PR, but I don't love this nested callbacks. I think most of these fetches are independent, if that's the case we can use `Promise.all` instead to have them happening in parallel. It can be a tecnical debt card for later, cc @ddelpiano